v3ElectronicsUserGuide.md

```diff
@@ -1323,8 +1323,12 @@ You can write a 24 bit mask to `GEM_AMC.OH.OHX.FPGA.TRIG.CTRL.VFAT_MASK` to mask
 
 The OH has the FPGA core temperature accessible from the sysmon registers in the OHv3 address table and nine PT100 sensors located around the board.  These PT100 sensors are read by the SCA when monitoring is enabled, see [Voltage & Temperature Monitoring].
 
+SCA gives output in the ADC counts. For the details of how the conversions for temperature and voltages are done look [here](docs/SCA_temperature_voltage_unit_conversion.md)
+
 [Top](https://github.com/cms-gem-daq-project/sw_utils/blob/develop/v3ElectronicsUserGuide.md#table-of-contents)
 
+
+
```

Comment:
nit picking but can you remove these two stray white lines that have been added?